tests_e2e/orchestrator/scripts/uncompress.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python3
```

Comment:
This script runs using whatever Python is installed on the test VM, and that is not necessarily Python 3.